code/controllers/subsystem/monitor.dm

```diff
@@ -0,0 +1,131 @@
+SUBSYSTEM_DEF(monitor)
+	name = "Monitor"
+	init_order = INIT_ORDER_MONITOR
+	runlevels = RUNLEVEL_GAME
+	wait = FREQUENCY_STATE_CALCULATION
+	///The next world.time for wich the monitor subsystem refresh the state
+	var/scheduled = 0
+	///The current state
+	var/current_state = STATE_BALANCED
+	///The last state
+	var/last_state = STATE_BALANCED
+	///If we consider the state as a stalemate
+	var/stalemate = FALSE
+	///The current state points. Negative means xenos are winning, positive points correspond to marine winning
+	var/current_points = 0
+	///The number of time we had the same state consecutively
+	var/stale_counter = 0
+	///The number of humans on ground
+	var/human_on_ground = 0
+	///The number of humans being in either lz1 or lz2
+	var/human_in_FOB = 0
+	///The number of time most of humans are in FOB consecutively
+	var/humans_all_in_FOB_counter = 0
+	///TRUE if we detect a state of FOB hugging
+	var/FOB_hugging = FALSE
+	///TRUE if xenos have hijacked into ship. Disable the monitor
+	var/hijacked = FALSE
+	///List of all int stats
+	var/list/list/monitor_statistics[B17_USED]
```

Comment:
this should be a holder datum, this list will get convoluted fast, as can be seen by the linter errors that cropped up already

an example would be 

```
var/datum/monitor_data/monitor_data = new

/proc/italia()
    SSmonitor.monitor_data.spicy_meatballs++
```